<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>175</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cryptocontracts</string>
  </property>
  <property name="locale">
   <locale language="English" country="Germany"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pButtonSelect">
      <property name="text">
       <string>Select PDF...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pButtonSign1">
      <property name="text">
       <string>Signature 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pButtonSign2">
      <property name="text">
       <string>Signature 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pButtonAddToLayer">
      <property name="text">
       <string>Add to chain</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QProgressBar" name="progressBar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>0</height>
       </size>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="contextMenuPolicy">
    <enum>Qt::ActionsContextMenu</enum>
   </property>
  </widget>
  <action name="actionSingle_Party">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Single Party</string>
   </property>
  </action>
  <action name="actionDouble_Party">
   <property name="text">
    <string>Double Party</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>pButtonSign2</tabstop>
  <tabstop>pButtonAddToLayer</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
